l10n_it_fatturapa_out/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2014 Davide Corio
+# Copyright 2015-2016 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 Matteo Mircoli Openforce srls
+
+from odoo import fields, models, api
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    phone_electronic_invoice = fields.Char(
```

Comment:
Non conviene metterlo in `l10n_it_fatturapa` insieme a tutti gli altri?

Reply:
per me è uguale @eLBati  io avevo pensato di metterlo qua perchè  riguardava il problema dell'esportazione

Reply:
però se effettivamente tutti i dati stanno dall'altra parte in giornata vedo di correggere la pr

Reply:
@eLBati  ho fatto la modifica come mi ha chiesto ho spostato al dichiarazione del modulo e quindi messo anche il relativo xpath.
Dimmi pure se ora è ok